hyper/contrib.py

```diff
@@ -27,6 +27,7 @@ class HTTP20Adapter(HTTPAdapter):
     def __init__(self, *args, **kwargs):
         #: A mapping between HTTP netlocs and ``HTTP20Connection`` objects.
         self.connections = {}
+        self.force_proto = kwargs.get('force_proto')
```

Comment:
Rather than grab this out of `kwargs`, let's make it an explicit keyword argument.